package.json

```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "start": "node --env-file=.env index.js",
-    "devstart": "nodemon --env-file=.env index.js",
+    "start": "node index.js",
```

Comment:
Try to revert and understand why it doesn't work,
Make sure you're on node 20.17.0